<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>test_prjClass</class>
 <widget class="QMainWindow" name="test_prjClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>956</width>
    <height>806</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>test_prj</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>../../x86release/main.ico</normaloff>../../x86release/main.ico</iconset>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout_3">
    <item row="0" column="0">
     <widget class="QPushButton" name="pushButton_MenuFile">
      <property name="text">
       <string>打开</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QTabWidget" name="tabWidget_Main">
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>120</height>
       </size>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="tab">
       <attribute name="title">
        <string>主页</string>
       </attribute>
      </widget>
      <widget class="QWidget" name="tab_2">
       <attribute name="title">
        <string>查看</string>
       </attribute>
      </widget>
     </widget>
    </item>
    <item row="1" column="0" colspan="2">
     <widget class="QWidget" name="widget" native="true">
      <layout class="QGridLayout" name="gridLayout">
       <item row="2" column="1">
        <widget class="QPushButton" name="drawWidget">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>50</height>
          </size>
         </property>
         <property name="text">
          <string>Draw Widget</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QPushButton" name="notepad">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>50</height>
          </size>
         </property>
         <property name="text">
          <string>Notepad</string>
         </property>
        </widget>
       </item>
       <item row="0" column="0" rowspan="2" colspan="4">
        <widget class="QLabel" name="imgLabel">
         <property name="autoFillBackground">
          <bool>true</bool>
         </property>
         <property name="text">
          <string>This is imgLabel</string>
         </property>
         <property name="scaledContents">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="0" column="4" rowspan="3">
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <widget class="QPushButton" name="pushButton">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>45</height>
            </size>
           </property>
           <property name="text">
            <string>PushButton</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="viewDelegate">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>45</height>
            </size>
           </property>
           <property name="text">
            <string>ViewDelegate</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="qTable">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>45</height>
            </size>
           </property>
           <property name="text">
            <string>QTable</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton_FileExtract">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>45</height>
            </size>
           </property>
           <property name="text">
            <string>文件提取工具</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton_LanTool">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>45</height>
            </size>
           </property>
           <property name="text">
            <string>辅助翻译工具</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton_MsgWin">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>45</height>
            </size>
           </property>
           <property name="text">
            <string>信息窗口测试</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="2" column="3">
        <widget class="QPushButton" name="pushButton_TestBtn">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>50</height>
          </size>
         </property>
         <property name="text">
          <string>测试按钮</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QDockWidget" name="dockWidget">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="floating">
    <bool>false</bool>
   </property>
   <property name="allowedAreas">
    <set>Qt::AllDockWidgetAreas</set>
   </property>
   <property name="windowTitle">
    <string>Message Window</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>8</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents">
    <layout class="QGridLayout" name="gridLayout_2">
     <item row="0" column="0">
      <widget class="QTextBrowser" name="textBrowser"/>
     </item>
    </layout>
   </widget>
  </widget>
  <action name="actionNotepad">
   <property name="text">
    <string>Notepad</string>
   </property>
  </action>
  <action name="actionQTable">
   <property name="text">
    <string>QTable</string>
   </property>
  </action>
  <action name="actionDrawWidget">
   <property name="text">
    <string>DrawWidget</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>pushButton</sender>
   <signal>clicked()</signal>
   <receiver>test_prjClass</receiver>
   <slot>on_pushbutton_clicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>355</x>
     <y>174</y>
    </hint>
    <hint type="destinationlabel">
     <x>299</x>
     <y>199</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionNotepad</sender>
   <signal>triggered()</signal>
   <receiver>test_prjClass</receiver>
   <slot>on_notepad_clicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>212</x>
     <y>211</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionQTable</sender>
   <signal>triggered()</signal>
   <receiver>test_prjClass</receiver>
   <slot>on_qtable_clicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>212</x>
     <y>211</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>drawWidget</sender>
   <signal>clicked()</signal>
   <receiver>test_prjClass</receiver>
   <slot>on_drawwidget_clicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>258</x>
     <y>286</y>
    </hint>
    <hint type="destinationlabel">
     <x>212</x>
     <y>211</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>notepad</sender>
   <signal>clicked()</signal>
   <receiver>test_prjClass</receiver>
   <slot>on_notepad_clicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>96</x>
     <y>286</y>
    </hint>
    <hint type="destinationlabel">
     <x>212</x>
     <y>211</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionDrawWidget</sender>
   <signal>triggered()</signal>
   <receiver>test_prjClass</receiver>
   <slot>on_drawwidget_clicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>186</x>
     <y>163</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>qTable</sender>
   <signal>clicked()</signal>
   <receiver>test_prjClass</receiver>
   <slot>on_qtable_clicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>139</x>
     <y>275</y>
    </hint>
    <hint type="destinationlabel">
     <x>182</x>
     <y>155</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>viewDelegate</sender>
   <signal>clicked()</signal>
   <receiver>test_prjClass</receiver>
   <slot>on_viewdelegate_clicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>304</x>
     <y>275</y>
    </hint>
    <hint type="destinationlabel">
     <x>182</x>
     <y>155</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_FileExtract</sender>
   <signal>clicked()</signal>
   <receiver>test_prjClass</receiver>
   <slot>on_listview_clicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>338</x>
     <y>194</y>
    </hint>
    <hint type="destinationlabel">
     <x>353</x>
     <y>226</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_LanTool</sender>
   <signal>clicked()</signal>
   <receiver>test_prjClass</receiver>
   <slot>on_searchlist_clicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>323</x>
     <y>228</y>
    </hint>
    <hint type="destinationlabel">
     <x>360</x>
     <y>259</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>on_pushbutton_clicked()</slot>
  <slot>on_notepad_clicked()</slot>
  <slot>on_qtable_clicked()</slot>
  <slot>on_drawwidget_clicked()</slot>
  <slot>on_viewdelegate_clicked()</slot>
  <slot>on_listview_clicked()</slot>
  <slot>on_searchlist_clicked()</slot>
 </slots>
</ui>
